resources/user-agents/browsers/k-meleon/k-meleon.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "K-Meleon #MAJORVER#.#MINORVER#",
-  "versions": [74, "1.7", "1.6", "1.5", "1.1", 1, "0.9"],
+  "versions": [75, 74, "1.7", "1.6", "1.5", "1.1", 1, "0.9", "0.8"],
```

Comment:
I noticed that version 75 had come out "recently" (last year I think) so I bumped that up here. The 0.8 version was added to test an agent I saw in the files.